src/plugins/orangehrmClaimPlugin/Controller/AssignClaimController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Claim\Controller;
+
+use OrangeHRM\Admin\Service\PayGradeService;
+use OrangeHRM\Core\Controller\AbstractVueController;
+use OrangeHRM\Core\Traits\Auth\AuthUserTrait;
+use OrangeHRM\Core\Traits\ServiceContainerTrait;
+use OrangeHRM\Core\Vue\Component;
+use OrangeHRM\Core\Vue\Prop;
+use OrangeHRM\Framework\Http\Request;
+use OrangeHRM\Framework\Services;
+
+class AssignClaimController extends AbstractVueController
+{
+    use ServiceContainerTrait;
+    use AuthUserTrait;
+
+    /**
+     * @return PayGradeService
+     */
+    public function getPayGradeService(): PayGradeService
+    {
+        return $this->getContainer()->get(Services::PAY_GRADE_SERVICE);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function preRender(Request $request): void
+    {
+        $currencies = $this->getPayGradeService()->getCurrencyArray();
+        $component = new Component('assign-claim-request');
+        $component->addProp(new Prop('currencies', Prop::TYPE_ARRAY, $currencies));
+        $component->addProp(new Prop('emp-number', Prop::TYPE_NUMBER, $this->getAuthUser()->getEmpNumber()));
```

Comment:
```php
        $component->addProp(new Prop('emp-number', Prop::TYPE_NUMBER, $this->getAuthUser()->getEmpNumber()));
```
`emp-number` change this prop name and claim employee auto complete `emp-number` prop name